src/clarity-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -652,25 +648,23 @@
         }
 
         .column-switch-wrapper {
-            // Has a possible sibling in the .datagrid-foot-select.checkbox container
-            flex: 1 1 1rem;
+            // We need this container to fill the space between itself and the right side of the footer
+            flex: 1 1 100%;
 
+            &.active {
+                .column-toggle--action {
+                    color: $clr-action-blue;
+                }
+            }
             .column-toggle--action {
-
                 // I'm overriding .btn/.btn-link here but am not confident this is the correct way to do it.
                 min-width: 0.75rem;
                 padding-left: 0;
                 padding-right: 0;
 
-                clr-icon {
```

Comment:
Why is this removed?

Reply:
The component doesn't need to target the clr-icon element directly anymore. I just target the `.column-toggle--action` class and let it cascade down. I think I wrote it originally before I really understood how clr-icons worked wrt setting color.

Reply:
Awesome. Thanks!